orchestrator/test_runner/src/main.rs

```diff
@@ -79,6 +81,9 @@ const TOTAL_TIMEOUT: Duration = Duration::from_secs(300);
 // The config file location for hermes
 const HERMES_CONFIG: &str = "/gravity/tests/assets/ibc-relayer-config.toml";
 
+// Module accounts
+pub const FEE_COLLECTOR_ADDRESS: &str = "gravity17xpfvakm2amg962yls6f84z3kell8c5l5f3cw4";
```

Comment:
could you clarify is this the distribution module or which module? Also note that it's generated by hashing a string and encoding as bech32, in fact maybe we should derive it from scratch here for flexibility.

Reply:
It's a separate module account used by multiple modules. We can't use it reliably in the test so I'm going to remove it